sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_v1/checks.sql

```diff
@@ -1,5 +1,5 @@
 #fail
-{{ is_unique(["`date`", "country"], "country IS NOT NULL") }}
+{{ is_unique(["submission_date", "country"], "country IS NOT NULL") }}
```

Comment:
Does this happen when the client reports more than one country on the `first_seen_date` (7 days before)? If yes, would it make sense to use first_seen_date here?